packages/swirl-components/src/components/swirl-select/swirl-select.tsx

```diff
@@ -41,6 +41,7 @@ export class SwirlSelect implements SwirlFormInput<string[]> {
   @Prop() searchPlaceholder?: string;
   @Prop() selectId?: string = Math.round(Math.random() * 1000000).toString();
   @Prop() swirlAriaDescribedby?: string;
+  @Prop() tooltip?: string;
```

Comment:
**question:** Why is the prop required here? Isn't it always set on the form-control? Also i cannot see any change in storybook once i set it.

Reply:
You are right it is not required. At my first attempt to implement this I added it on every component which could be nested inside a form-control and not on the top level form-control.
I was sloppy by removing this code and missed that one.
Thanks :)